### ISSUE TITLE
Have `--remove-macros` accept multiple macros, or change it to `--remove-macro`

### ISSUE THREAD

**Issue by orrp** [sylvainhalle/textidote#255]
Thank you so much for this wonderful tool. I have a small suggestion as a new user.

Suppose there are three macros I would like TeXtidote to ignore: `mymacro0`, `mymacro1` and `mymacro2`.

Because the `--remove-macros` option refers to macros in the plural, I would expect it to be used as follows:
```bash
textidote --remove-macros mymacro1 mymacro2 mymacro3 main.tex
```

However, this does not work and instead I must invoke textidote with:
```bash
textidote --remove-macros mymacro1 --remove-macros mymacro2 --remove-macros mymacro3 main.tex
```

I suggest modifying command line parsing so that `--remove-macros` accepts a list of macros as in the first snippet above. Note that if the list is of length one we recover the current behavior, so this is not a breaking change. An easier fix would be to rename the option to `--remove-macro`, although I think this is less desirable from a usability standpoint.

**Comment by orrp** [comment]
Update: I just noticed #250, which is related to the above issue (I missed it because the title refers to environments rather than macros). I seems that specifying multiple commas via a comma-separated list is supported. However, this is not mentioned in the `--help` documentation.

Accepting a space-delimited list of macros is more consistent with UNIX command line tools, so I would still suggest changing the behavior to be as in the first code snippet in my original post, and updating the documentation in `--help` to say something of the sort:
```
--remove-macros mac-1 ... mac-n   Remove LaTeX macros specified by a space-delimited list.
```

**Comment by sylvainhalle** [comment]
I agree. This will require an update to the underlying command line parser; an issue is already open for this: https://github.com/sylvainhalle/Bullwinkle/issues/17
